src/extras/CoolPubSubClient.h

```diff
@@ -15,6 +15,7 @@
 #define MQTT_VERSION_3_1      3
 #define MQTT_VERSION_3_1_1    4
 
```

Comment:
Don't change this file at all please - it's a 3rd party library and our coding standards don't apply here.